build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt

```diff
@@ -85,8 +85,44 @@ private fun Project.registerRootProjectArtifactSizeMetricsTask(
 }
 
 open class ArtifactSizeMetricsPluginConfig {
+    /**
+     * Changes the prefix used to get artifact size metrics in the
+     * "collectDelegatedArtifactSizeMetrics" task. For developer use only
```

Comment:
nit: I'd say this whole repository is "For developer use only", the comment is kind of unnecessary